src/Bundle/StorageBundle/Field/Types/FileFieldType.php

```diff
@@ -93,6 +93,7 @@ function getFormOptions(FieldableField $field): array
             'endpoint' => $this->generateEndpoint($field->getSettings()),
             'upload-sign-url' => $url,
             'upload-sign-csrf-token' => $this->csrfTokenManager->getToken('pre_sign_form'),
+            'acl' => $field->getSettings()->bucket['acl'],
```

Comment:
acl bucket settting is not always there (fe. in the unit tests)

`'acl' => $field->getSettings()->bucket['acl'] ?? ''`